hack/generator/azure-arm.yaml

```diff
@@ -141,6 +141,41 @@ typeTransformers:
     remove: true
     because: it has no writable properties in swagger
 
+  - name: "*"
+    ifType:
```

Comment:
I'd prefer `ifType` to go after `property` here as that is the way it is applied